frontend/package.json

```
@@ -21,7 +21,7 @@
     "@types/react": "17.0.5",
     "@types/styled-components": "5.1.9",
     "@types/yup": "0.29.11",
-    "axios": "0.21.1",
+    "axios": "^0.24.0",
```

```suggestion
    "axios": "0.24.0",
```

We are not using ^ in versions (we are fixing the version on the package.json. This might be discussed and changed in another PR, but I would recommend not doing it here, because that would make the package.json inconsistent.